### PR TITLE
Do not directly update the validationState and use the /draft endpoint

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#109

### CODE REVIEW

File: client/src/app/metadata-details-dialog/metadata-details-dialog.component.ts [hunk 1]
```diff
@@ -96,7 +100,7 @@ export class MetadataDetailsDialogComponent implements OnInit {
         this.dialogRef.close();
       }, err => {
         this.alertService.clear();
-        this.alertService.success('Error',
+        this.alertService.error('Error',
```
Comment (jacobwindsor) [inline client/src/app/metadata-details-dialog/metadata-details-dialog.component.ts:103]: :D

File: client/src/app/shared/components/data-table/data-table.component.spec.ts [hunk 2]
```diff
@@ -18,7 +34,7 @@ describe('DataTableComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(DataTableComponent);
     component = fixture.componentInstance;
-    component.rows = [{}];
+    component.dataSource = dataSource;
```
Comment (jacobwindsor) [inline client/src/app/shared/components/data-table/data-table.component.spec.ts:37]: thanks for fixing the test!

File: client/src/app/shared/services/ingest.service.ts [hunk 3]
```diff
@@ -273,20 +272,22 @@ export class IngestService {
...
+  public patch<T>(ingestLink, patchData): Observable<T>;
+  public patch<T>(ingestLink, patchData): Observable<T> {
+    return this.http.patch<T>(ingestLink, patchData);
   }
 
-  public get(url, options?): Observable<Object> {
-    return this.http.get(url, options);
-  }
-
-  public getAs<T>(url): Observable<T> {
```
Comment (jacobwindsor) [inline client/src/app/shared/services/ingest.service.ts:288]: Not that it's a problem but why remove getAs?

**Review by jacobwindsor — APPROVED** [review]
Great work :)

**Comment by aaclan-ebi** [comment]
Thanks for the review, Jacob!

**Pull request merged**